packages/evolution-common/src/services/parsers/ParserFunction.ts

```diff
@@ -0,0 +1,14 @@
+import { UserInterviewAttributes, InterviewResponses, Household, Person } from '../interviews/interview';
+
+export type ParserFunction<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> = (
```

Comment:
Please add a comment what will this function be used for. It receives in parameter the personsById AND the personsArray, which are redundant, and a single Person, and a Household, which should contain the personsById directly, so I'm not sure all these parameters are required